ims/company/lost_or_stolen.md

```diff
@@ -0,0 +1,24 @@
+# What To Do When Your Device is Lost or Stolen
+
+The first thing you should do if your device is lost or stolen is report the incident to the CSO or IMS Manager as soon as possible.
+
+For Team members using the standard build of Made Tech device, we are able to use MDM software to remotely wipe data from your device. 
+
+Those who are not using a Made Tech issued device should consult the [BYOD policy.](byod.md)
```

Comment:
A very very very minor point, excuse my pedanticness, but I'd move the full stop outside of the link.